app/src/hooks/useAuth.tsx

```diff
@@ -49,6 +49,26 @@ const useProvideAuth = () => {
     await deleteStoredToken();
   };
 
+  const initiatePasswordReset = async (email: string) => {
+    await axios.post('auth/forgot-password', {
+      email,
+    });
```

Comment:
Zwróćmy Promise z Axiosa żeby można było obsłużyć błędy w componencie

```suggestion
    return axios.post('auth/forgot-password', {
      email,
    });
```